src/middleware/errorHandler.js

```diff
@@ -182,6 +183,9 @@ const handleValidationError = (error, req, res, next) => {
 		// sanitize
 		const err = new PageNotFound(error);
 		next(err);
+	} else if (error instanceof OpenApiValidator.error.BadRequest) {
+		const err = new BadRequest(error);
+		next(err);
 	} else {
 		next(error);
```

Comment:
I think we can move next(err) outside if statements as it will execute always.

Reply:
please note it is throwing a newly constructed error in both cases and the original error in the else-case